src/DynamoPublish/ViewModels/PublishViewModel.cs

```diff
@@ -184,13 +220,6 @@ private bool CanPublish(object obj)
                 return false;
             }
 
-            if (String.IsNullOrWhiteSpace(Description))
-            {
-                UploadStateMessage = Resources.ProvideWorskspaceDescriptionMessage;
-                IsReadyToUpload = false;
-                return false;
-            }
```

Comment:
This removes the need to provide a description for a customizer - it's not required.